frontend/src/metabase/admin/tools/components/ToolsApp.tsx

```diff
@@ -4,7 +4,7 @@ import {
   AdminNavItem,
   type AdminNavItemProps,
   AdminNavWrapper,
-} from "metabase/admin/settings/components/AdminNav";
+} from "metabase/admin/AdminNav";
```

Comment:
since these nav components are used by several admin sections, not just settings, I moved them up a level